page/ajax/jquery-ajax-methods.md

```diff
@@ -30,32 +30,32 @@ $.ajax({
 
 	// The type of data we expect back
 	dataType : "json",
+})
+  // Code to run if the request succeeds (is done);
+  // The response is passed to the function
+  .done(function( json ) {
+     $( "<h1>" ).text( json.title ).appendTo( "body" );
+     $( "<div class=\"content\">").html( json.html ).appendTo( "body" );
+  })
+  // Code to run if the request fails; the raw request and
+  // status codes are passed to the function
+  .fail(function( xhr, status, errorThrown ) {
+    alert( "Sorry, there was a problem!" );
+    console.log( "Error: " + errorThrown );
+    console.log( "Status: " + status );
+    console.dir( xhr );
+  })
+  // Code to run regardless of success or failure;
+  .always(function( xhr, status ) {
+    alert( "The request is complete!" );
+  });
 
-	// Code to run if the request succeeds;
-	// the response is passed to the function
-	success: function( json ) {
-		$( "<h1>" ).text( json.title ).appendTo( "body" );
-		$( "<div class=\"content\">").html( json.html ).appendTo( "body" );
-	},
-
-	// Code to run if the request fails; the raw request and
-	// status codes are passed to the function
-	error: function( xhr, status, errorThrown ) {
-		alert( "Sorry, there was a problem!" );
-		console.log( "Error: " + errorThrown );
-		console.log( "Status: " + status );
-		console.dir( xhr );
-	},
-
-	// Code to run regardless of success or failure
-	complete: function( xhr, status ) {
-		alert( "The request is complete!" );
-	}
-});
 ```
 
 **Note:** A note about the `dataType` setting: if the server sends back data that is in a different format than you specify, your code may fail, and the reason will not always be clear, because the HTTP response code will not show an error. When working with Ajax requests, make sure your server is sending back the data type you're asking for, and verify that the `Content-type` header is accurate for the data type. For example, for JSON data, the `Content-type` header should be `application/json`.
 
+**Deprecation:** Deprecation Notice: The jqXHR.done(), jqXHR.fail(), and jqXHR.always() callbacks are used above, but you may see the older jqXHR.success(), jqXHR.error(), and jqXHR.complete() callbacks, and should update the code accordingly: As of jQuery 1.8 these are deprecated and will eventually be removed. 
```

Comment:
This is more something for API docs IMHO. Keep in mind that most people are here to learn and use api.jquery.com as a reference. I feel like we shouldn't bother them with deprecated stuff they should be aware of just yet.